src/main/scala/com/microsoft/hyperspace/index/sources/default/DefaultFileBasedSource.scala

```diff
@@ -72,8 +73,20 @@ class DefaultFileBasedSource(private val spark: SparkSession) extends FileBasedS
           Hdfs.Properties(Content.fromLeafFiles(files, fileIdTracker).get)
         val fileFormatName = fileFormat.asInstanceOf[DataSourceRegister].shortName
 
+        // Use case-sensitive map if the provided options are case insensitive.
+        val caseSensitiveOptions = options match {
```

Comment:
`val caseSensitiveOptions: Map[String, String] = `
Could you add type here instead of below `opt`?  And it seems spark always uses the case insensitive map but this would be ok just in case.

nit: `case _ => _ `

Reply:
thanks @sezruby , removed the type. 

> nit: `case _ => _`

could you please explain what does this mean?